sketch.js

```diff
@@ -12,64 +18,80 @@ for (var i=0; i <= 16; i++){
   var yPosition = 550;
   var rectWidth = 5;
   var datayear= -data[i]/120;
-  
+
   var check = collidePointCircle(mouseX,mouseY,153, 470, 10, 10); //see if the mouse is in the rect
 
 	if(check){ //change color!
-		text("90 Million",130,450);
+		words["text"] = "90 Million";
```

Comment:
Rather than trying to render every text every time, reuse some of the same variables you've already established.